docs/docs-beta/docs/dagster-plus/deployment/code-locations/code-location-history.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Code location history"
+displayed_sidebar: "dagsterPlus"
+sidebar_position: 4
+sidebar_label: "Code location history"
+---
+
+# Code location history
+
+Dagster+ automatically tracks metadata every time a code location is loaded. This can be used to understand when changes have been made, and what those changes were. In addition, this metadata can be used to quickly redeploy an older version.
+
+## Viewing code location history
+
+1. In the Dagster+ UI, navigate to the **Deployment** tab.
+2. In the row associated with the code location you're interested in, click **View history** on the **Updated** column.
+
+![Screenshot highlighting the "Updated" column for a code location)](/img/placeholder.svg)
+
+This will bring up a modal showing a history of every time that code location has been loaded, and metadata associated with that load.
+
+If you have connected Dagster+ to a GitHub or Gitlab repository, each row will have a link to the commit that was deployed at that point in time.
+
+If a code location has been deployed multiple times with identical metadata, these rows will be collapsed together. You can expand them by deselecting **Collapse similar entries** in the top left corner of the modal.
+
+
+## Rolling back to a previous code location version
+
+If you notice an issue with newly deployed code, or your code fails to deploy successfully, you can quickly roll back to a previously deployed image that's known to work properly.
+
+1. In the Dagster+ UI, navigate to the **Deployment** tab.
+2. In the row associated with the code location you're interested in, click **View history** on the **Updated** column.
+3. In the dropdown menu next to **View metadata**, select **Rollback to this version**.
```

Comment:
```suggestion
3. In the **Actions** column click the dropdown menu to the right of **View metadata**, select **Rollback to this version**.
```

I will say this two-in-one style of button is slightly unintuitive, at least when trying to describe it.